DuckDB.NET.Data/DuckDBCommand.cs

```diff
@@ -8,10 +8,12 @@ namespace DuckDB.NET.Data;
 
 public class DuckDbCommand : DbCommand
 {
-    private DuckDBConnection connection;
+    private DuckDBConnection? connection;
     private readonly DuckDBParameterCollection parameters = new();
 
+#pragma warning disable CS8765 // Nullability of type of parameter doesn't match overridden member (possibly because of nullability attributes).
```

Comment:
Is it possible to avoid these pragmas and match the base nullability?

Reply:
Yes, it sounds possible. Will need some additional changes but manageable.